cluster-service/Makefile

```diff
@@ -41,7 +40,7 @@ deploy:
 	  --set tenantId=$${TENANT_ID} \
 	  --set region=${REGION} \
 	  --set serviceKeyvaultName=${SERVICE_KV} \
-	  --set csServicePrincipalCredsBase64=$${CS_SERVICE_PRINCIPAL_CREDS_BASE64} \
+	  --set csServicePrincipalCredsBase64="eyAidGVuYW50SWQiOiAiIiwgImNsaWVudElkIjogIiIsICJjbGllbnRTZWNyZXQiOiAiIiB9Cg==" \
```

Comment:
Just unset json?  Could we move this to the helm template?

Reply:
this will be removed soon alltogether but needs a patch in CS.
for now CS reads the content of the secret, verifies that the expected fields are there and then ignores it

@miguelsorianod PTAL at this